### PR TITLE
Add std version output for agent.

### PR DESCRIPTION
**Pull request by BalaBalaYi**
### What changes were proposed in this pull request?

Printout the dlrover's version at the beginning when worker started.

### Why are the changes needed?

For better maintenance.

### Does this PR introduce any user-facing change?

No.

### How was this patch tested?

UT.

### CODE REVIEW

**Review by majieyue — APPROVED** [review]
lgtm

File: dlrover/python/util/common_util.py [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+
+def get_installed_version(package_name):
+    """
+    Get the installed version of a package.
+
+    Args:
+        package_name (str): The name of the package.
+
+    Return:
+        result: Version of the package.
```
Comment (samplise) [inline dlrover/python/util/common_util.py:52]: It is better to include the return value type.

File: dlrover/python/util/file_util.py [hunk 2]
```diff
@@ -27,3 +28,35 @@ def is_same_path(path1: str, path2: str):
...
+    """
+    Find target file in parent directories.
+
+    Args:
+        filename (str): Target filename.
+        start_dir (str, optional): Target directory. Defaults to None(from
+            current directory).
+
+    Returns:
+        result: The target file path.
```
Comment (samplise) [inline dlrover/python/util/file_util.py:43]: The same as above.

Comment (BalaBalaYi) [reply dlrover/python/util/file_util.py:46]: done

File: dlrover/python/util/common_util.py (hunk 1 above)
Comment (BalaBalaYi) [reply dlrover/python/util/common_util.py:52]: done

**Pull request merged**